cypress/integration/test/contact.spec.js

```diff
@@ -0,0 +1,37 @@
+describe("Contact api testing", () => {
+    it("GET", () => {
+  
+      cy.request({
+        method: "GET",
+        url: "http://localhost:3000/contacts",
+      }).then((response) => {
+  
+        expect(response.status).equal(200);
+        expect(response.body).to.not.be.null;
```

Comment:
Cambiar el expect(X).to.not.be.null por un expext(X).to.deep.equal(X) es decir, no testear por lo que no es, sino por lo que se espera.

Reply:
Corregido en el commit "Contact testing was modified"